### DISCUSSION TITLE
How to get a url to an object in storage that doesnt expire?

### DISCUSSION THREAD

**Discussion by Aging-Developer** [supabase/supabase#15290] [Questions]
Is there a way to have an object url not to expire in storage:
![image](https://github.com/supabase/supabase/assets/122821259/f4faff78-3cc4-47c3-a80e-17853ab71302)
seems totally crazy that cannot, that would mean iI have to continuously monitor for content vanishing from my app!

**Comment by GaryAustin1** [comment]
You just use a public URL with a public bucket.   A public bucket is exactly the same as a private bucket except for the option for a public URL.  You can even make your public URL hard to guess with a UUID.
